IOS_SYMBOLICATION.md

```diff
@@ -0,0 +1,76 @@
+# Symbolicating iOS crash reports
+
+Debugging an iOS application crash sometimes involves analyzing crash reports.
+More info about crash reports can be found
+[in the official documentation](https://developer.apple.com/library/archive/technotes/tn2151/_index.html).
+
+Crash reports generally require symbolication to become properly readable:
+symbolication turns raw hex addresses into human-readable source locations.
+The document below describes some specific details of symbolicating crash reports
+from iOS applications using Kotlin.
+
+## Enable .dSYM for release Kotlin binaries
+
+To symbolicate addresses in Kotlin code (e.g. for stack trace elements
+corresponding to Kotlin code) `.dSYM` bundle for Kotlin code is required.
+
+By default Kotlin/Native compiler doesn't produce `.dSYM` for release
+(i.e. optimized) binaries. This can be changed with `-Xg0` experimental
+compiler flag: it enables debug info and `.dSYM` bundle generation for produced
+release binaries. To enable it in Gradle, use
+
+```kotlin
+this.targets.forEach {
+    if (it is KotlinNativeTarget) {
+        it.compilations.all {
+            kotlinOptions {
+                freeCompilerArgs += "-Xg0"
+            }
+        }
+    }
+}
+```
+
+(in Kotlin DSL).
```

Comment:
Maybe give Groovy DSL example as well?

Reply:
This would make the document more cluttered with Gradle snippets.
Generally `kotlin-native`-hosted documentation seems to feature only Kotlin DSL.

Reply:
OK